packages/wonder-blocks-popover/components/popover-content-core.js

```diff
@@ -101,7 +109,7 @@ const styles = StyleSheet.create({
             Colors.offBlack8
         }`,
         margin: 0,
-        maxWidth: 280,
+        maxWidth: 288,
```

Comment:
Should we be using our `Spacing` constants to build these values rather than magic numbers?

Reply:
Good idea! it's great to see the design spec is following the spacing constraints :)